src/style.css

```diff
@@ -1 +1,100 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 450;
+  
+}
+
+body {
+
+  margin: 0;
+
+}
+
+ul {
+
+  margin: 0;
+  
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.image {
+
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__item {
+  
+  display: flex;
+  align-items: center;
+  margin: 0 5px;
+  padding: 0 5px;
```

Comment:
for spacing elements either use padding or margin, i don't see why use them both